exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  if (sentence.includes("code")) {
```

Comment:
Awesome, great job Shima.

Like I shared earlier in the PR, you could simplify this without the if statement.

```javascript
function containsCode(sentence) {
  return (sentence.includes("code"));
}
```